packages/admin/cms-admin/src/warnings/warningConfig.ts

```diff
@@ -0,0 +1,14 @@
+import { type ReactNode } from "react";
+
+import { useCometConfig } from "../config/CometConfigContext";
+import { warningMessages as cometWarningMessages } from "./warningMessages";
+
+export interface WarningConfig {
+    warningMessages: Record<string, ReactNode>;
```

Comment:
We could remove the warning prefix – what do you think?

Reply:
I was thinking that if there are more features, it might need more options and so you could add non-breaking changes easily, but it is probably the wrong approach to just prepare for breaking changes. without even knowing what could be added here. So I'll remove it yes

Reply:
Having an object for the config is a good idea 👍🏼 I was just thinking that `<CometConfigProvider  warnings={{ warningMessages: ... }} />` could be replaced with `<CometConfigProvider  warnings={{ messages: ... }} />`

Reply:
ahh okay, makes sense I'll do that 👍

Reply:
changed here: da0e362997fe1df2204f59c5734c5b1f42f9c314